internal/server/rule_manager.go

```diff
@@ -256,8 +256,7 @@ func stopRuleInternal(name string) {
 	}
 }
 
-func stopRule(name string) (result string) {
-	var err error
+func stopRule(name string) (err error, result string) {
```

Comment:
error should be the last return valur

Reply:
can you take a look at the failed test case , not understand why

Reply:
looks like a random error. I just rerun it.